package.json

```diff
@@ -10,7 +10,8 @@
     "react-scripts": "1.1.4",
     "redux": "^4.0.1",
     "redux-thunk": "^2.3.0",
-    "tslib": "^1.9.3"
+    "tslib": "^1.9.3",
+    "typescript": "^3.2.4"
   },
   "scripts": {
     "start": "react-scripts start",
```

Comment:
Remember that when migrating to typescript you need a transpiler in order to run the code in the browser. I've neve converted a .js project to ts, but I think you could get away with using react-scripts-ts instead of react-scripts.